client/src/components/InteractionCard/InteractionCard.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import InteractionCard from "./InteractionCard";
+import { checkPropTypes } from "prop-types";
+
+const handleClick = e => console.log("clicked");
+
+storiesOf("InteractionCard", module).add("Default", () => (
+  <InteractionCard link="/route" handleClick={handleClick} />
```

Comment:
Can you call the `handleClick` prop `onClick` and the `link` prop `to`? That will be more inline with convention. The Button prop should be called `onClick` too